src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order_checkout.yml

```diff
@@ -63,3 +63,7 @@ winzou_state_machine:
                     on: ["complete"]
                     do: ["@sylius.state_resolver.order_shipping", "resolve"]
                     args: ["object"]
+                sylius_set_order_immutable_names:
```

Comment:
We should move it to the main order state machine. This is not really specific to checkout, it should happen whenver we create a new order (cart -> new transition).